src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -1292,9 +1313,26 @@ void VM_Version::get_processor_features() {
     }
           _features =   CPUFeatures_x64;
     _glibc_features = GLIBCFeatures_x64;
+
+    if (ShowCPUFeatures) {
+      if (ignore_glibc_not_using) {
+	tty->print_cr("CPU features are being kept intact as requested by -XX:CPUFeatures=ignore");
```

Comment:
Could you check whitespace error https://github.com/openjdk/crac/pull/41/checks?check_run_id=13867663789

Reply:
Sorry, I have provided some local reconfigurations so that it hopefully should not happen again.